serializations.md

```diff
@@ -214,8 +214,19 @@ subsequent sections divided by object.
 | Channel | 58 |
 | Channel snapshot transaction | 59 |
 | POI | 60 |
+| Trees with DB | 61 |
```

Comment:
This goes according to [serialization](https://github.com/aeternity/epoch/blob/master/apps/aecore/src/aec_object_serialization.erl#L76-L85)